### PR TITLE
neovim-bare(-nightly): Add manifests

### PR DESCRIPTION
**Pull request by rashil2000**
These manifests decouple Neovim from the heavily outdated [neovim-qt](https://github.com/equalsraf/neovim-qt) GUI client.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by issaclin32** [comment]
Hash check is failing for `neovim-bare-nightly` even after I ran checkver locally.
We should probably use `"version: nightly"` to avoid errors.

File: bucket/neovim-bare-nightly.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "version": "0.7.0-dev-812-g2f31e7b88",
+    "description": "Vim fork focused on extensibility and usability",
+    "homepage": "https://neovim.io",
+    "license": {
+        "identifier": "Apache-2.0,Vim",
+        "url": "https://github.com/neovim/neovim/blob/master/LICENSE"
+    },
+    "suggest": {
+        "vcredist": "extras/vcredist2015"
```
Comment (issaclin32) [inline bucket/neovim-bare-nightly.json:10]:
```suggestion
        "vcredist": "extras/vcredist2022"
```
VCRedist 2022 covers the functionality of VCRedist 2015-2022. See https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170

**Comment by rashil2000** [comment]
That's because they just made a nightly release 8 minutes ago 😅

image

Even neovim-nightly in master branch has not updated. It will get updated in around 3 hours.

File: bucket/neovim-bare.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+    "version": "0.6.1",
+    "description": "Vim-fork focused on extensibility and usability",
+    "homepage": "https://neovim.io/",
+    "license": {
+        "identifier": "Apache-2.0,Vim",
+        "url": "https://github.com/neovim/neovim/blob/master/LICENSE"
+    },
+    "suggest": {
+        "vcredist": "extras/vcredist2015"
```
Comment (issaclin32) [inline bucket/neovim-bare.json:10]:
```suggestion
        "vcredist": "extras/vcredist2022"
```

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**